Containerfile

```diff
@@ -4,6 +4,10 @@ LABEL maintainer="LSIT Systems <lsitops@ucsb.edu>"
 
 USER root
 
+RUN apt-get update && \
```

Comment:
I don't think this will matter and it might actually be more proper to do it this way, but most images currently use `apt` instead of `apt-get` for update/upgrade/install.  `apt-get` has to be used for clean.  `apt` is generally faster but "less stable" so generates a warning but I've yet to see a case where `apt` didn't work and `apt-get` did.

Reply:
You/we might want to change that in the pstat215 image too then cause I was lazy and did a copy/paste from that repo for this.